lib/ejs.js

```diff
@@ -170,7 +171,11 @@ function includeFile(path, options) {
   if (!opts.filename) {
     throw new Error('`include` requires the \'filename\' option.');
   }
-  opts.filename = exports.resolveInclude(path, opts.filename);
+  if(path.substr(0,1) === "/" && opts.root){
+    opts.filename = exports.resolveInclude(path.replace(/^\/*/,""), opts.root, 1);
+  }else{
```

Comment:
Please follow the style guide and put spaces before and after braces. The same applies to line 174.